content/applications/productivity/iot/config/connect.rst

```diff
@@ -1,81 +1,143 @@
-      should become
-      **http://375228-saas-11-5-iot-f3f920-all.runbot16.odoo.com\|4957098401**).
+   .. note::
+      If the :abbr:`IoT (Internet of Things)` Box WiFi connection wizard doesn't start navigate to
+      the :abbr:`IoT (Internet of Things)` Box admin console page by entering the  
 
-#. Choose the WiFi network you want to connect with (enter the password
-   if there is one) and click on Submit. Wait a few seconds before being
-   redirected to your database.
+#. Choose the WiFi network that the :abbr:`IoT (Internet of Things)` box will connect with (enter
+   the password if there is one) and click on :guilabel:`Connect`. Wait a few seconds before being
+   redirected to the database. The computer may need to be manually connected back to WiFi.
 
-   .. image:: connect/connect08.png
+   .. image:: connect/configure-wifi-network-iot.png
+      :align: center
+      :alt: Configuring the WiFi for the IoT box.
 
-You should now see the IoT Box.
+The :abbr:`IoT (Internet of Things)` box should appear in the :menuselection:`IoT` module of the
+Odoo database.
 
-.. image:: connect/connect09.png
+.. image:: connect/iot-box-connected.png
    :align: center
+   :alt: The IoT box has been successfully configured on the Odoo database.
+
+.. important::
+   A manual reset of the :abbr:`IoT (Internet of Things)` box may be necessary upon successfully
+   connecting the :abbr:`IoT (Internet of Things)` box via WiFi for the :abbr:`IoT
+   Internet of Things)` box to appear in the :menuselection:`IoT` module of the Odoo database.
+
+.. _iot_connect/token:
+
+Manually connecting the IoT via the token
+=========================================
+
+A manually connection of the :abbr:`IoT (Internet of Things)` box to the :menuselection:`IoT
+(Internet of Things)` module can be made with the :guilabel:`token`. The :guilabel:`token` can be
+found after clicking on :guilabel:`Connect` in the :menuselection:`IoT` module. The
+:guilabel:`token` will be inputted into the :abbr:`IoT (Internet of Things)` box admin console.
+
+#. Access the :abbr:`IoT (Internet of Things)` box admin console by entering the :abbr:`IP
+   (Internet Protocol)` of the :abbr:`IoT (Internet of Things)` box into a browser window.
+
+   .. note::
+   The :abbr:`IP (Internet Protocol)` can be accessed by the router the :abbr:`IoT
+   (Internet of Things)` box is connected to or by connecting a printer to the :abbr:`IoT
+   (Internet of Things)` box (a receipt will print out with the :abbr:`IoT (Internet of Things)`
+   box's :abbr:`IP (Internet Protocol)` on it).
+
+#. Enter the :guilabel:`token` under the :guilabel:`Server` section by clicking on
+   :guilabel:`Configure`.
+
+#. Paste the :guilabel:`token` into the :guilabel:`Server Token` field and click
+   :guilabel:`Connect`.
 
 IoT Box Schema
 ==============
 
-.. image:: connect/connect10.png
-   :align: center
+Raspberry Pi 4
+--------------
+
+.. figure:: connect/iot-box-schema.png
```

Comment:
nice!
However, as the 4 USBs used 4 connection dots on the schema, technically we should also use 2 connections dots for the 2 micro HDMI for consistency